frontend/src/metabase/ui/components/buttons/Button/Button.styled.tsx

```diff
@@ -158,13 +159,13 @@ export const getButtonOverrides = (): MantineThemeOverride["components"] => ({
 });
 
 const getPrimaryColor = (theme: MantineTheme, colorName: string) => {
-  return theme.fn.themeColor(colorName, theme.fn.primaryShade());
```

Comment:
`theme.fn.primaryShade()` doesn't do anything here, since we pass the same shade to Mantine even if each color require 10 shades.